internal/unix/types_linux.go

```diff
@@ -12,4 +12,5 @@ const (
 	AF_UNSPEC         = linux.AF_UNSPEC
 	AF_INET           = linux.AF_INET
 	AF_INET6          = linux.AF_INET6
+	AF_BRIDGE         = lunix.AF_BRIDGE
```

Comment:
just a typo:
```suggestion
	AF_BRIDGE         = linux.AF_BRIDGE
```

Reply:
thanks, taken care

Reply:
I cannot merge by myself, could you please help to merge?